fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -460,11 +460,13 @@ private Response getBinaryContent(final String rangeValue, final FedoraResource
         }
 
     private RdfStream getTriples(final FedoraResource resource, final Set<? extends TripleCategory> x) {
-        return resource.getTriples(translator(), x);
+        // TODO Reimplement filtering of triples by categories
+        return resource.getTriples();
```

Comment:
I am just getting here with `POST` and I am wondering if we don't want to pass in a translator instance to convert `info:fedora/the/ldp/path` to `http://localhost:8080/fcrepo/rest/the/ldp/path` in the RDFStream?

Reply:
I was thinking this might happen as another step in the RDF stream just prior to returning to clients, versus occurring when getting the base triples. This might allow us to transform the server managed and client managed properties at together.

Reply:
What was your thinking for the old `RdfContexts`? It seems like instead of `getTriples(*, PROPERTIES)` now we will `getTriples().filter(PROPERTIES)`. Should we do something to RdfStream to setup some filters in there? Or are we going to cache these RdfStreams or are they _only_ containing the requested information. 

These are just questions, not comments on your work here. 👍

Reply:
The `RdfContext`s sort of did a combination of filtering out and generating new triples. My thinking was that that logic would be moved into separate services which would do similar stuff. These services would be implemented/called in the places where they are needed rather than inside the FedoraResource class.

For instance, there could be a `InboundReferencesRdfService` which concats the inbound refs as rdf to the triples rdf stream.

It may also still be reasonable to keep the RdfContexts structure and just externalize the gluing together of all of them, but we're working against a pretty different model than the node and property API from modeshape, so I don't know if it'll end up being practical.

Reply:
So this `resource.getTriples()` just returns the user's RDF and nothing else? Every other type of triple (containment, membership, reference, server managed) will be add (or not) to the stream via these services?

Reply:
Yeah pretty much. Server managed triples get added in at the HTTP layer via a service that calls getManagedProperties (and probably generates some other properties that are not stored) and concats that onto the original Triples stream. I'm not sure if its a good thing or not for the rdfStream returned by getTriples() to be modified or if it should always only return the triples that were returned by persistence.